src/librustc_mir/monomorphize/polymorphize.rs

```diff
@@ -28,7 +28,7 @@ pub fn provide(providers: &mut Providers) {
 /// Determine which generic parameters are used by the function/method/closure represented by
 /// `def_id`. Returns a bitset where bits representing unused parameters are set (`is_empty`
 /// indicates all parameters are used).
-fn unused_generic_params(tcx: TyCtxt<'_>, def_id: DefId) -> FiniteBitSet<u64> {
+fn unused_generic_params(tcx: TyCtxt<'_>, def_id: DefId) -> FiniteBitSet<u32> {
```

Comment:
Can we expect that the if the return value is relevant we already called that method in the local crate?

If so, it might be more performant to add
```rust
if !def_id.is_local() {
    return FiniteBitSet::new_empty();
}
```

Reply:
Fixed!